omod/src/main/webapp/resources/scripts/htmlForm.js

```diff
@@ -40,6 +40,26 @@
         return jq(".submitButton.confirm").is(":disabled");
     }
 
+
+    /**
+     * Change the hours, minutes and seconds dropdowns to new date with client timezone
+     * also change the encounterDate datepicker current date to new date with client timezone
+     **/
+    var changeTimeWidgetClientTimeZone = function (clientDateTime) {
+        getField('encounterDate.value').datepicker('setDate', clientDateTime)
+        jq("#encounterDate").find(".hfe-hours").val(clientDateTime.getHours()).change();
+        jq("#encounterDate").find(".hfe-minutes").val(clientDateTime.getMinutes()).change();
+        jq("#encounterDate").find(".hfe-seconds").val(clientDateTime.getSeconds()).change();
+    }
+
+    /**
+     * Receives a datetime on ISO8601 format and return only the date part.
+     * Example: receives the datetime 2021-04-01T15:00:00.000+0000, the output is 04-01-2021.
+     **/
+    var splitDate = function (date) {
+        return moment(date.split('T'))[0].toDate()
```

Comment:
Did you test the above and confirm that it runs?

Reply:
I tested and it works

Reply:
What does the `moment` function return?

Reply:
Lets imagine this scenario: 
date = "2021-04-01T23:59:59.000+0300"
moment(date.split('T')[0]).toDate()
It will return a date without time and with client timezone-> Thu Apr 01 2021 00:00:00 GMT+0100
This will work for what we want.
 
The other way to do this is like I was doing with a custom function.

Reply:
Are you really sure that you tested line 60 above and confirmed that it runs? 
Do you notice the difference between line 60 and the equivalent that you have just put in your comment?

Reply:
The code we are commenting is outdated, that one : `moment(date.split('T'))[0].toDate()` have a problem with the position of the `[0]`

Reply:
> The code we are commenting is outdated, that one : `moment(date.split('T'))[0].toDate()` have a problem with the position of the `[0]`

cc dkayiwa

Reply:
hi @dkayiwa  since we need this so as we can release htmlformentryui,does it look good to be merged now

Reply:
Have all the comments been addressed?

Reply:
> Have all the comments been addressed?

hi @icrc-jfigueiredo   thanks for all the works on this, could you kindly attend to all comments from @dkayiwa  since this is needed so as we make a release, thanks

Reply:
> > Have all the comments been addressed?
> 
> hi @icrc-jfigueiredo thanks for all the works on this, could you kindly attend to all comments from @dkayiwa since this is needed so as we make a release, thanks

Yes they were. Everything is ok. CC @HerbertYiga  @dkayiwa

Reply:
@icrc-jfigueiredo take a second look.

Reply:
I answered to all of the posts. Did I miss something?

Reply:
Did you carefully go through all the comments and confirm that you committed everything that you were supposed to?

Reply:
I just did that. If there are any new questions, please let me know. thank you.

Reply:
> I just did that. If there are any new questions, please let me know. thank you.

cc @dkayiwa